geoextent/lib/extent.py

```diff
@@ -34,6 +34,18 @@ def compute_bbox_wgs84(module, path):
         raise Exception("The bounding box could not be transformed to the target CRS epsg:{} \n error {}"
                         .format(hf.WGS84_EPSG_ID, e))
 
+    validate = hf.validate_bbox_wgs84(spatial_extent['bbox'])
+    logger.debug("Validate: {}".format(validate))
+
+    if not hf.validate_bbox_wgs84(spatial_extent['bbox']):
+        try:
+            flip_bbox = hf.flip_bbox(spatial_extent['bbox'])
+            spatial_extent['bbox'] = flip_bbox
```

Comment:
validate_bbox_wgs84 only checks that the coordinates are within [WGS84 bounds](https://epsg.io/4326) (e.g latitude values between -90 and 90) 

`flip_bbox` tries to flip the bbox and then validates the new flipped bbox. In case the new bbox is not validated the function raise an exception.